tinkerpop3/src/main/java/io/shiftleft/overflowdb/storage/OdbStorage.java

```diff
@@ -95,10 +98,11 @@ public void close() {
     closed = true;
     logger.info("closing " + getClass().getSimpleName());
     if (mvstore != null) mvstore.close();
+    if (!doPersist) mvstoreFile.delete();
```

Comment:
wondering if this should be maybe in try/catch?

Reply:
if the tmp file can't get deleted, closing the storage is incomplete and we should escalate that up. 
what would you suggest we do in the catch block?